style.css

```diff
@@ -139,29 +139,99 @@ h2 {
 
 /* Video */
 .wsuwp-youtube-wrap-outer {
+	background: #fff;
 	bottom: -2rem;
 	left: 50%;
 	position: absolute;
 	width: 50vw;
 }
 
-.wuswp-youtube-wrap-inner {
+.wsuwp-youtube-wrap-outer:before {
 	border: 2rem solid #981e32;
-	background: #fff;
-	padding-bottom: calc(56.25% - 4rem);
+	box-sizing: border-box;
+	content: "";
+	display: block;
+	height: 100%;
+	left: 0;
+	position: absolute;
+	top: 0;
+	width: 100%;
+}
+
+.wsuwp-youtube-wrap-outer:after {
+	color: #fff;
+	content: "Welcome to WSU";
+	float: left;
+	font-size: .9rem;
+	font-weight: 600;
+	letter-spacing: 3px;
+	margin: -4.5rem 0 0 3.5rem;
+	pointer-events: none;
 	position: relative;
+	text-transform: uppercase;
 }
 
-.wuswp-youtube-wrap-inner iframe {
-	height: calc(100% + 4rem);
-	left: -2rem;
+.video-in-scroll {
+	bottom: 1rem;
+	left: auto;
+	position: fixed;
+	right: 1rem;
+	width: 30vw;
+	z-index: 99;
+}
+
+.wuswp-youtube-wrap-inner {
+	background: url(https://why.wsu.edu/wp-content/uploads/sites/1552/2016/12/img-video-placeholder.jpg) center no-repeat #fff;
```

Comment:
It'd be nice to pull this image into the repo itself so that it's not reliant on the site ID, but it's not a big deal.